src/platform_impl/web/web_sys/resize_scaling.rs

```diff
@@ -127,7 +127,7 @@ impl ResizeScaleInternal {
              (-webkit-device-pixel-ratio: {current_scale})",
         );
         let mql = MediaQueryListHandle::new(window, &media_query, closure);
-        assert!(
+        debug_assert!(
```

Comment:
Doesn't this mean that the bug is still here on Debug mode?

Reply:
Yes, I should probably remove the changelog entry now that I think about it.

For context: it's not really a problem, because `privacy.resistFingerprinting` isn't stable in Firefox and has to be enabled through `about:config`. This is in addition to being a Firefox bug, Winit does everything right here.